apiary.apib

```diff
@@ -475,29 +477,32 @@ Get details for a single invoice.
                             + product (object, nullable)
                                 + type: `product` (string)
                                 + id: `e2314517-3cab-4aa9-8471-450e73449040` (string)
-                            + quantity: `3` (number)
+                            + quantity: 3 (number)
                             + description: `An awesome product` (string)
                             + unit_price (object)
                                 + Include Money
                                 + tax: `excluding` (enum[string])
                                     + Members
                                         + excluding
                             + tax (object)
-                                + rate: `0.21` (number)
+                                + type: `taxRate` (string)
+                                + id: `c0c03f1e-77e3-402c-a713-30ea1c585823` (string)
                             + discount (object, nullable)
                                 + type: `percentage` (enum[string])
                                     + Members
                                         + percentage
-                                + value: `15.00` (number)
+                                + value: 15.00 (number)
             + total (object)
                 + tax_exclusive (Money)
                 + tax_inclusive (Money)
                 + payable (Money)
                 + taxes (array)
                     + (object)
-                        + rate: `0.21` (number)
-                        + taxable (Money)
-                        + tax (Money)
+                        + tax (object)
+                            + type: `taxRate`
+                            + id: `c0c03f1e-77e3-402c-a713-30ea1c585823` (string)
```

Comment:
Why do we need tax (type, id) in each of the line items and also in the total? Is is a default tax for the invoice?